manim/grpc/impl/frame_server_impl.py

```diff
@@ -10,250 +10,279 @@
 import subprocess as sp
 import threading
 import time
+import traceback
 import ctypes
 from ...utils.module_ops import (
     get_module,
     get_scene_classes_from_module,
     get_scenes_to_render,
+    scene_classes_from_file,
 )
 from ... import logger
 from ...constants import JS_RENDERER_INFO
+from ...renderer.js_renderer import JsRenderer
+from ...utils.family import extract_mobject_family_members
+import logging
+import copy
+from ...mobject.value_tracker import ValueTracker
+from ...mobject.types.vectorized_mobject import VMobject
+from ...mobject.types.image_mobject import ImageMobject
 
 
-class FrameServer(frameserver_pb2_grpc.FrameServerServicer):
-    def animation_index_is_cached(self, animation_index):
-        return animation_index < len(self.keyframes)
+class MyEventHandler(FileSystemEventHandler):
```

Comment:
Why is it named as `MyEventHandler`? Could be named anything else IMO than having something like this in a library.

Reply:
Noted